modules/test-utils/README.md

```diff
@@ -0,0 +1,56 @@
+# @luma.gl/test-utils
```

Comment:
I see the value in having info in the npm readme page, but In general I would avoid duplicating information, in the past similar setups have quickly gotten out of sync. A short prologue and then linking to the docs is the right approach IMHO.